include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -6356,7 +6359,7 @@ namespace VectorTools
 
       // append result of this cell to the end of the vector
       AssertIsFinite(diff);
-      return diff;
+      return diff; // TODO: what about diff_mean? Not necessarily double
```

Comment:
Yes, that works. I think I simply didn't understand from that one sentence what the TODO was about. In either case, document your choice.